admin_scripts/dconf_policy_desktop.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+#================================================================
+#% SYNOPSIS
+#+    dconf_policy_desktop.sh [FILE]
+#%
+#% DESCRIPTION
+#%    This script changes and locks the desktop background for all users on the
+#%    system using a dconf lock.
+#%
+#%    It takes one optional parameter: the path to the desktop background. If
+#%    this parameter is missing or empty, any existing background lock will be
+#%    removed.
+#%
+#================================================================
+#- IMPLEMENTATION
+#-    version         dconf_policy_desktop.sh (magenta.dk) 1.0.0
+#-    author          Alexander Faithfull
+#-    copyright       Copyright 2019, Magenta ApS
+#-    license         GNU General Public License
+#-    email           af@magenta.dk
+#-
+#================================================================
+#  HISTORY
+#     2019/09/24 : af : Script created
+#
+#================================================================
+# END_OF_HEADER
+#================================================================
+
+set -x
+
+POLICY="/etc/dconf/db/os2borgerpc.d/00-background"
+POLICY_LOCK="/etc/dconf/db/os2borgerpc.d/locks/background"
+
+if [ "$1" = "" ]; then
+    rm -f "$POLICY"
+    rm -f "$POLICY_LOCK"
+else
+    if [ ! -d "`dirname "$POLICY"`" ]; then
+        mkdir "`dirname "$POLICY"`"
+    fi
+    if [ ! -d "`dirname "$POLICY_LOCK"`" ]; then
+        mkdir "`dirname "$POLICY_LOCK"`"
+    fi
+
+    cat > "/etc/dconf/profile/user" <<END
```

Comment:
More documentations here...

Reply:
Addressed in commit b0375d2241d133a862fe038b2ba2cec028490f3a.